test-common/uk/gov/hmrc/alcoholdutyaccount/common/TestData.scala

```diff
@@ -224,13 +234,299 @@ trait TestData extends ModelGenerators {
         items = Seq(
           FinancialTransactionItem(
             subItem = "001",
+            dueDate = None,
             amount = 1000.00
           )
         )
       )
     )
   )
 
+  val singleFullyOutstandingReturn: FinancialTransactionDocument =
+    FinancialTransactionDocument(
+      financialTransactions = Seq(
+        FinancialTransaction(
+          sapDocumentNumber = sapDocumentNumberGen.sample.get,
+          periodKey = Some(periodKey),
+          chargeReference = Some(chargeReferenceGen.sample.get),
+          originalAmount = BigDecimal("9000"),
+          mainTransaction = TransactionType.toMainTransactionType(TransactionType.Return),
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("9000")),
+          items = Seq(
+            FinancialTransactionItem(
+              subItem = "000",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("9000")
+            )
+          )
+        )
+      )
+    )
+
+  val singlePartiallyOutstandingReturn: FinancialTransactionDocument =
```

Comment:
instead of duplicating all these values, could you use copy on standard vals and change the necessary bits. Will be easier to read and maintain

Reply:
Future refactoring